### PR TITLE
A page to create products

### PR DESCRIPTION
**Pull request by anujSingh-A-writer**
JIRA: https://anujsingh.atlassian.net/browse/NEX-22

### CODE REVIEW

File: components/form/Form.tsx
Comment (Comaecod) [inline components/form/Form.tsx:1]: Delete this file.

File: pages/products/create.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    useState<Partial<ProductDetails>>();
+
+  const handleCreate: React.FormEventHandler<HTMLFormElement> = async (
+    e: React.FormEvent<HTMLFormElement>
+  ) => {
+    // 👇️ prevent page refresh
+    e.preventDefault();
+
+    // CALL CREATE API
+    await axios.post(CREATE_PRODUCT, productDetails).then(() => {
```
Comment (Comaecod) [inline pages/products/create.tsx]: Create a hook for API requests.

**Review by Comaecod — CHANGES_REQUESTED** [review]
Few comments

File: components/form/Form.tsx
Comment (anujSingh-A-writer) [reply components/form/Form.tsx:1]: Deleted

File: pages/products/create.tsx (hunk 1 above)
Comment (anujSingh-A-writer) [reply pages/products/create.tsx]: Learning SWR, a react hook.

**Review by Comaecod — APPROVED** [review]
Looks Good!

**Pull request merged**